analyzer-models/src/main/java/ffb/analyzer/models/espn/Player.java

```diff
@@ -0,0 +1,177 @@
+package ffb.analyzer.models.espn;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import ffb.analyzer.models.espn.deserializers.PlayerRankingsDeserializer;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ * Entity representing a player.
+ */
+public class Player extends EspnEntity<Player> {
+
+    /**
+     * Enum for a player's position.
+     */
+    public enum Position {
+        QB(1),
+        RB(2),
+        WR(3),
+        TE(4),
+        K(5),
+        DEFENSE(16);
+
+        private static final Map<Integer, Position> POSITION_BY_ID;
+
+        static {
+            POSITION_BY_ID = List.of(Position.values())
+                .stream()
+                .collect(Collectors.toMap(Position::getValue, Function.identity()));
+        }
+
+        private final int id;
+
+        /**
+         * Converts an integer to a {@link Position}.
+         * @param positionId ID of the position.
+         * @return {@link Position}.
+         */
+        public static Position valueOf(int positionId) {
+            return POSITION_BY_ID.get(positionId);
+        }
+
+        Position(Integer id) {
+            this.id = id;
+        }
+
+        public int getValue() {
+            return id;
+        }
+    }
+
+    /**
+     * Enum for a player's injury status.
+     */
+    public enum InjuryStatus {
+        NORMAL,
+        ACTIVE,
+        QUESTIONABLE,
+        OUT,
+        DOUBTFUL
+    }
+
+    private boolean active;
+    private Position position;
+    private boolean droppable;
+    private List<Integer> eligibleSlots;
+    private String firstName;
+    private String fullName;
```

Comment:
@pecjas Thoughts on pulling full name in? We can get the same info from `firstName` and `lastName`.